### PR TITLE
chore(starters): add gatsby-starter-banshorian

### PR DESCRIPTION
**Pull request by webmaeistro**

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -6235,3 +6235,21 @@
     - Sitemap Generation
     - XML Sitemaps
     - Progressive Web Ap
+- url: https://gatsby-starter-banshorian.now.sh
```
Comment (iamdarshshah) [inline docs/starters.yml:6238]: You can redeploy your site on **Vercel** server by importing your repo. here _`https://vercel.com/import/git`_.

ZEIT's Now is changed to **Vercel** _`https://vercel.com/`_ and after deploying your project on Vercel server, your url will look similar to _`https://gatsby-starter-banshorian.vercel.app`_.

Thanks for raising PR.

Comment (webmaeistro) [reply docs/starters.yml:6238]: I just did exactly that the URL's vercel spits back at me is: 
DOMAINS
gatsby-starter-banshorian.webmaeistro.now.sh
gatsby-starter-banshorian.now.sh
gatsby-starter-banshorian-git-master.webmaeistro.now.sh

gatsby-starter-banshorian-now-sh

Comment (iamdarshshah) [reply docs/starters.yml:6238]: Okay, it seems deployment is done using Vercel CLI and Vercel is still using `.now.sh` after deployment.

Refer this `https://vercel.com` as it showcases that the output link should end with `.vercel.app`

Comment (webmaeistro) [reply docs/starters.yml]: I also have a gatsby-starter-banshorian.netlify.app URL . I could opted for that instead to make it easy..? :D

Comment (iamdarshshah) [reply docs/starters.yml]: LGTM! Thanks

**Comment by webmaeistro** [comment]
ran lint, ran prettier & changed URL to 'gatsby-starter-banshorian.netlify.app'

**Comment by webmaeistro** [comment]
@iamdarshshah Appreciate it! 😎

**Comment by webmaeistro** [comment]
@iamdarshshah - why is it blocked when i got 1 approving review? 
image

**Comment by iamdarshshah** [comment]
@webmaeistro merging pull request is done by the core team members. They will get back to you and review it. You should keep it open for getting reviewed by the team if you want.

Thanks

**Pull request merged**